idseq_dag/steps/build_custom_blast_index.py

```diff
@@ -8,7 +8,28 @@
 import idseq_dag.util.count as count
 
 class PipelineStepBuildCustomBlastIndex(PipelineStep):
-    ''' Build Custom Blast Index implementation '''
+    """ From GSNAP, we know the *best match* from the NT db, for each read. 
+    All sequences that have accessions in the list of NT matches are downloaded. 
+    A BLAST database is generated from the resulting .fa file via the following command:
```

Comment:
add single quotes to .fa -> '.fa' or something else of indicate the difference to a normal period.
WE can also called it fasta file.